app/build.gradle

```diff
@@ -29,13 +29,13 @@ repositories {
 
 android {
 
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    buildToolsVersion rootProject.ext.buildToolsVersion
+    compileSdkVersion BuildVersions.compileSdkVersion
+    buildToolsVersion BuildVersions.buildToolsVersion
 
     defaultConfig {
         applicationId "com.faskn.app.weatherapp"
```

Comment:
Also maybe  we can create a file which is named as Configs.kt under buildSrc and this class can hold packageName, compileSdkVersion etc.

Reply:
Done. 9620035